src/sentry/integrations/opsgenie/client.py

```diff
@@ -116,9 +117,12 @@ def build_issue_alert_payload(
 
     def send_notification(self, data):
         headers = self._get_auth_headers()
-        with record_event(OnCallInteractionType.CREATE).capture():
-            resp = self.post("/alerts", data=data, headers=headers)
-        return resp
+        with record_event(OnCallInteractionType.CREATE).capture() as lifecycle:
+            try:
+                return self.post("/alerts", data=data, headers=headers)
+            except (ApiUnauthorized, ApiError) as e:
```

Comment:
`ApiUnauthorized` is a subclass of `ApiError`. if you're already catching `ApiError` i don't think you need to catch `ApiAuthorized`